Lib/fontTools/ttLib/tables/S_V_G_.py

```diff
@@ -16,44 +16,46 @@
 __doc__="""
 Compiles/decompiles version 0 and 1 SVG tables from/to XML.
 
-Version 1 is the first SVG definition, implemented in Mozilla before Aug 2013, now deprecated.
-This module will decompile this correctly, but will compile a version 1 table
-only if you add the secret element "<version1/>" to the SVG element in the TTF file.
+Version 1 is the first SVG definition, implemented in Mozilla before Aug 2013,
```

Comment:
“Aug” should be “August”.

Reply:
Will fix later, but I haven't got that far in systematically going through the classes yet. This was just fixing a RST syntax nit.